resources/carrier/en/44.txt

```diff
@@ -13,6 +13,9 @@
 # limitations under the License.
 
 # Carrier details updated from http://www.ofcom.org.uk/static/numbering/S7.xls
+# Carrier names have been updated to friendlier trading/brand names
+# The mapping of company names can be found in the project below.
```

Comment:
95?!
I guess we should still do it, they are only comments and will be stripped when built anyway, and it's good to verify them in one place.